lib/macho/macho_file.rb

```diff
@@ -556,7 +560,7 @@ def low_fileoff
 
       segments.each do |seg|
         sections(seg).each do |sect|
-          next if sect.size == 0
+          next if sect.size == 0 # rubocop:disable Style/ZeroLengthPredicate
```

Comment:
Looks like RuboCop still complains here about a `Style/NumericPredicate` violation. I'd personally just prefer `zero?` here, since it's equally readable (IMO, of course) and we don't have to disable two checks in-line.